src/main/resources/liquibase/2.7.x.x/base.xml

```diff
@@ -7,4 +7,7 @@
 
 	<include file="liquibase/2.7.x.x/address_line.xml"/>
 	<include file="liquibase/2.7.x.x/geographical_unit.xml"/>
+	<include file="liquibase/2.7.x.x/localization.xml"/>
```

Comment:
@caseyi is this slated for 2.7 or should it be merged in to develop which is still at 2.6?